tests/test_flashflood.py

```diff
@@ -152,7 +152,7 @@ def _test_replay(self, expected_events, number_of_retries=5):
                         del expected_events[event.event_id]
             if not expected_events:
                 return
-            time.sleep(0.5)
+            time.sleep(1.0)
```

Comment:
![Screen Shot 2019-11-06 at 11 28 28 AM](https://user-images.githubusercontent.com/1947534/68330759-a448ad00-0088-11ea-8fd2-740e9e6486dc.png) you think i would miss this?

Reply:
Things progressed more quickly than I anticipated 😆